client/web/src/site-admin/backend.ts

```diff
@@ -898,7 +907,7 @@ export const SITE_EXTERNAL_SERVICE_CONFIG = gql`
     }
 `
 
-export const WEBHOOKS = gql`
+const WebhookFieldsFragment = gql`
```

Comment:
we usually make query constants all uppercase and use underscores
```suggestion
const WEBHOOK_FIELDS_FRAGMENT = gql`
```